blacklist

```diff
@@ -6,6 +6,7 @@
   "AitorDB/home-assistant-sun-card",
   "alryaz/hass-component-yandex-smart-home",
   "alryaz/hass-mosenergosbyt",
+  "alryaz/hass-yandex-music-browser",
```

Comment:
This is not enough, please open an issue here to have it removed <https://github.com/hacs/integration/issues/new?assignees=ludeeus&labels=flag&projects=&template=removal.yml>